plugins/fishlim/plugin_hexchat.c

```diff
@@ -138,7 +138,7 @@ static int handle_outgoing(char *word[], char *word_eol[], void *userdata) {
 static int handle_incoming(char *word[], char *word_eol[], hexchat_event_attrs *attrs, void *userdata) {
     const char *prefix;
     const char *command;
-    const char *recipient;
+    const char *channel = hexchat_get_info(ph, "channel");
```

Comment:
Yes, you get it from the message contents like it did before. Perhaps it was grabbing the wrong word or something?